Arrangement_on_surface_2/test/Arrangement_on_surface_2/Traits_base_test.h

```diff
@@ -123,8 +122,8 @@ class Traits_base_test : public IO_test<Geom_traits_T> {
     typename Traits::Equal_2 equal = this->m_geom_traits.equal_2_object();
     if (equal(exp_answer, real_answer)) return true;
 
-    std::string exp_answer_str = boost::lexical_cast<std::string>(exp_answer);
-    std::string real_answer_str = boost::lexical_cast<std::string>(real_answer);
+    std::string exp_answer_str = std::to_string(exp_answer);
```

Comment:
Is `Point_2` guaranteed to be an alias for an integer type in this code?

Reply:
Oh, I forgot to check that. I probably only meant to apply this to the lexical_casts of integers.

Reply:
Maybe it makes more sense to do:

```c++
std::ostringstream os_exp_answer << exp_answer;
std::ostringstream os_real_answer << real_answer;
this->print_answer(os_exp_answer.str(), os_real_answer.str(), "point");
```

But I will just leave the code as it was.